packages/extras/src/lib/interactivity/setupInteractivity.ts

```diff
@@ -7,6 +7,26 @@ function getIntersectionId(event: Intersection) {
   return `${(event.eventObject || event.object).uuid}/${event.index}${event.instanceId}`
 }
 
+/**
+ * Dispatches a synthetic pointermove event for touchmove interactions
+ * because the browser may not send pointermove events for touch, depending
+ * on the canvas touch-action css property.
+ */
+function transformTouchMove(event: TouchEvent) {
+  const touch = event.touches[0]
+  if (!touch || !event.target) {
+    return
+  }
+  const syntheticPointerEvent = new PointerEvent('pointermove', {
+    clientX: touch.clientX,
+    clientY: touch.clientY,
+    bubbles: event.bubbles,
+    cancelable: event.cancelable,
+    pointerType: 'touch'
+  })
+  event.target.dispatchEvent(syntheticPointerEvent)
```

Comment:
Why dispatch again instead of just passing the synthetic pointermove event along in place of the native touch event inside the event handler? The interactivity code depends on offsetX/offsetY properties, but these cannot be set via JS; the event has to be dispatched to cause them to be calculated and set